src/Telemetry/Microsoft.DotNet.AzureDevOpsTimelineTests/BuildLogScraperTests.cs

```diff
@@ -84,6 +84,21 @@ public async Task BuildLogScraperShouldExtractOneESHostedPoolImageName()
             Assert.AreEqual(MockAzureClient.MicrosoftHostedAgentImageName, imageName);
         }
 
+        [Test]
+        public async Task BuildLogScraperShouldExtractDockerImageName()
+        {
+            CancellationTokenSource cancellationTokenSource = new CancellationTokenSource();
```

Comment:
No need to create a source. You can create a token directly or you can use `CancellationToken.None`

Reply:
Replaced the token with `CancellationToken.None` everywhere